src/utils/aiprdescription/openai.ts

```diff
@@ -52,14 +52,13 @@ export const generateDescription = async (
                     },
                 ],
                 temperature,
-                max_tokens: 600,
                 stream: false,
                 n: 1,
             },
         );
 
         return completion;
     } catch (error: any) {
-        alert(error);
+        console.log("OpenAI error: ", error.message);
```

Comment:
consider using `console.error` for these cases

Reply:
Did it here https://github.com/open-sauced/browser-extensions/pull/79/commits/9147a422ec6d290916dcedb47af12fac026bdb86.